src/collector/top_score_collector.rs

```diff
@@ -811,6 +881,11 @@ where
                 return;
             }
         }
+
+        if !self.acceptor.accept(&doc) {
+            return;
+        }
```

Comment:
All that boilerplate just for this!  haha

Reply:
😅